apple/internal/partials/framework_import.bzl

```diff
@@ -284,6 +286,10 @@ def _generate_symbols(
             if not sets.contains(has_dsym_framework_basenames, framework_basename):
                 all_binaries.append(framework_binary)
 
+    # Skip the action if there're no binaries that can provide symbols
+    if not all_binaries:
+        return []
```

Comment:
This fixes a bug - previously the action would still be executed (with just `mkdir -p /path/to/symbols_files`) even if the binary list was empty.